src/main/java/io/confluent/connect/jdbc/sink/BufferedRecords.java

```diff
@@ -232,6 +236,9 @@ private long executeDeletes() throws SQLException {
     long totalDeleteCount = 0;
```

Comment:
nit (same as above): signature needs to include throws SQLException, ConnectException